src/components/modules/blog/code/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Roboto_Mono } from 'next/font/google';
+
+import { StyledBlogCodeContent, StyledBlogCodeTitleWrapper, StyledBlogCodeWrapper } from './code.styles';
+import type { BlogCodeProps } from './code.types';
+
+import { Text } from '@/components/elements/text';
+
+const dmCodeFont = Roboto_Mono({ subsets: ['latin'], weight: ['400'], variable: '--font-code' });
+
+export const BlogCode = ({ title, content }: BlogCodeProps) => {
+  return (
+    <StyledBlogCodeWrapper className={dmCodeFont.className}>
+      <StyledBlogCodeTitleWrapper>
+        <Text fontSize="m" fontWeight="bold" color="white">
+          {title}
+        </Text>
+      </StyledBlogCodeTitleWrapper>
+      <StyledBlogCodeContent>{content}</StyledBlogCodeContent>
```

Comment:
Probably the client client will want to handle different code languages. Can we confirm it and adapt it if needed?

Reply:
I don't think it would be the case we will need to update the component since it has no highlights, so don't know if the answer will make no difference, we could ask about if he wants text hightlights on the code. What do you think about it?